docs/reference/pages/panels.md

```diff
@@ -109,6 +109,44 @@ Here are some built-in panel types that you can use in your panel definitions. T
 
 ```
 
+### JavaScript customizations in the admin  
+
+You may want to execute some JavaScript when an inline panel items are ready, added or removed, The ``w-formset:ready``, ``w-formset:added`` and ``w-formset:removed`` events allow this. 
```

Comment:
No need for double backticks, that's only required in some of the older RST formatted docs.